azure-pipelines.yml

```diff
@@ -6,3 +6,15 @@ trigger:
 
 jobs:
 - template: job--pre-commit.yml
+- template: job--python-tox.yml
+  parameters:
+    toxenvs:
+      - py27
+      - py34
+      - py35
+      - py36
+      - py37
+      - py38
+      - pypy2
+      - pypy3
+      - docs
```

Comment:
this'll need `coverage: false` -- I also prefer the inline list here for `toxenvs: [py27, py34, ...]`